### ISSUE TITLE
V0.8.5 with Homey V2 and multi mode does not work properly

### ISSUE THREAD

**Issue by EdwardDumay** [ArjanKranenburg/virtual-devices#87]
In Homey V1 it work very good. Just swipping to another mode made it work. Now I see 4 buttons with not the text of the modes. So you don't know what you are doing.

**Comment by ArjanKranenburg** [comment]
Thanks for reporting. The technique I used for this is is no longer supported in V2. I'm still battling with Athom (see also issue #77 and https://github.com/athombv/homey-apps-sdk-issues/issues/33)

So I'll probably have to find a new solution, but frankly don't feel like putting more energy into something that Athom may change again in V3 or so.

**Comment by liteautomation1** [comment]
oh damn, i too was waiting for a fix for this. It is one of the best virtual devices, and I use it for a HDMI matrix. With no individual icon settings we just dont know which button is which. In the old app it was great. 
ps I think v3 would be some time away.

**Comment by giggo** [comment]
Was there a change in v3 that would allow this to work?

**Comment by ArjanKranenburg** [comment]
No, I mentioned V3 above to express my discontent of Athom not being backwards compatible across versions. 
I did not see any changes in V3 that restored the lost functionality.
